src/stan-users-guide/parallelization.Rmd

```diff
@@ -687,3 +699,62 @@ function if they are rectangular.
 If the outputs are of varying sizes, then there will have to be some way
 to convert it back to a usable form based on the input, because there
 is no way to directly return sizes or a ragged structure.
+
+## OpenCL
+
+OpenCL (Open Computing Language) is a framework that enables writing programs that
+execute across heterogeneous platforms. An OpenCL program can be run on CPUs, GPUs,
+DSPs or FPGAs. In order to run OpenCL programs, an OpenCL runtime be installed on
```

Comment:
Has anyone run Stan on a DSP or FPGA? If not let's just say OpenCL lets Stan run on GPUs.

Reply:
I have a labmate experimenting with FPGA and some basic Stan code, but yeah, I agree.

Reply:
Oh interesting. That's more of a feature for us as OpenCL users tho -- as user facing docs, there isn't a supported way to run Stan on your FPGA as of now.

Reply:
Indeed.